gradle/publish.gradle

```diff
@@ -178,6 +181,17 @@ subprojects {
   }
   check.dependsOn('checkPom')
 
+  task updateExpectedPom(dependsOn: generatePomFileForMavenPublication, type: Copy) {
+    description 'After having verified changes with checkPom, this task will perform the copy.'
+
+    def expectedPomFile = sourceSets.test.resources.find() { it.name == "expected-pom.xml" }
```

Comment:
Why does the expected file need to exist in order to update/place. We could use this task to generate the baseline for a newly-minted module.

Reply:
I suppose it doesn't!